plugins/org.python.pydev.debug/src/org/python/pydev/debug/ui/launching/PythonRunnerConfig.java

```diff
@@ -533,6 +538,30 @@ public PythonRunnerConfig(ILaunchConfiguration conf, String mode, String run,
         this.pythonpathUsed = p;
     }
 
+    private static List<List<Tuple<String, String>>> getPathMappingsList()
+            throws MisconfigurationException {
+        try {
+            List<List<Tuple<String, String>>> pathMappings = new ArrayList<List<Tuple<String, String>>>();
+
+            JsonArray pathMappingsJSON = JsonArray.readFrom(PydevPrefs.getEclipsePreferences()
+                    .get(PyDevEditorPreferences.PATHS_FROM_ECLIPSE_TO_PYTHON,
+                            PyDevEditorPreferences.DEFAULT_PATHS_FROM_ECLIPSE_TO_PYTHON));
+
+            for (JsonValue jsonValue : pathMappingsJSON) {
+                List<Tuple<String, String>> tempList = new ArrayList<Tuple<String, String>>();
+                JsonObject jsonObject = jsonValue.asObject();
+                List<String> keys = jsonObject.names();
+                for (String key : keys) {
+                    tempList.add(new Tuple<String, String>(key, jsonObject.get(key).asString()));
+                }
+                pathMappings.add(tempList);
+            }
+            return pathMappings;
+        } catch (Exception e) {
+            throw new MisconfigurationException("Error parsing Path Mappings config JSON\n\nDetails:\n" + e);
```

Comment:
We shouldn't fail here, rather, just log it instead of failing.

Something as:

`Log.log(e)`.